cvmfs/libcvmfs_int.cc

```diff
@@ -282,7 +284,8 @@ cvmfs_context* cvmfs_context::Create(const options &opts) {
   cvmfs_context *ctx = new cvmfs_context(opts);
   assert(ctx != NULL);
 
-  if (ctx->Setup(opts) != 0) {
+  perf::Statistics *statistics = new perf::Statistics(); // provisional
```

Comment:
No, it isn't. I mean, it is in the end of the program. It is just a workaround to avoid clashes between different repositories (contexts) mounted at the same time.